dags/dap_collector_ppa_prod.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.cncf.kubernetes.secret import Secret
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+DOCS = """
+### PPA Prod DAP Collector
+
+#### Description
+
+Runs a Docker image that collects PPA Prod Environment data from a DAP (Distributed Aggregation Protocol) leader and stores it in BigQuery.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/dap-collector-ppa-prod)
+
+This DAG requires following variables to be defined in Airflow:
+* dap_ppa_prod_auth_token
+* dap_ppa_prod_hpke_private_key
+* dap_ppa_prod_task_config_url
+* dap_ppa_prod_ad_config_url
+
+This job is under active development, occasional failures are expected.
+
+#### Owner
+
+bbirdsong@mozilla.com
+"""
+
+default_args = {
+    "owner": "bbirdsong@mozilla.com",
+    "email": ["ads-eng@mozilla.com", "bbirdsong@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 6, 26),
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 0,
+}
+
+project_id = "moz-fx-ads-prod"
+ad_table_id = "ppa.measurements"
+report_table_id = "ppa.reports"
+
+tags = [
+    Tag.ImpactTier.tier_3,
+    Tag.Triage.no_triage,
+]
+
+hpke_private_key = Secret(
+    deploy_type="env",
+    deploy_target="HPKE_PRIVATE_KEY",
+    secret="airflow-gke-secrets",
+    key="DAP_PPA_PROD_HPKE_PRIVATE_KEY",
+)
+
+auth_token = Secret(
+    deploy_type="env",
+    deploy_target="AUTH_TOKEN",
+    secret="airflow-gke-secrets",
+    key="DAP_PPA_PROD_AUTH_TOKEN",
+)
```

Comment:
Switched the secrets, but may need to tweak the `key` params here once the values are configured in GSM.